tests/integration/fixtures/pgtest/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.9"
+services:
+  db:
+    container_name: buildingmotif-db
+    image: postgres
+    environment:
+      - "POSTGRES_PASSWORD=password"
+      - "POSTGRES_USER=bmotif_pgtest"
+      - "POSTGRES_DB=bmotif_pgtest"
+    ports:
+      - target: 5432
+        published: 5432
+        protocol: tcp
+        mode: host
+    healthcheck:
+      test: ["CMD", "pg_isready", "-d", "postgresql://bmotif_pgtest:password@db:5432/bmotif_pgtest"]
+      interval: 10s
+      timeout: 30s
+      retries: 5
+      start_period: 5s
+
+  api:
+    container_name: buildingmotif-api
+    environment:
+      - "DB_URI=postgresql://bmotif_pgtest:password@db:5432/bmotif_pgtest"
+    depends_on: # require that the db is up and listening
+      db:
+        condition: service_healthy
+    build:
+      context: ../../../../.
+      dockerfile: buildingmotif/api/Dockerfile
+    image: buildingmotif-api:latest
+    ports:
+      - target: 5000
+        published: 5000
+        protocol: tcp
+        mode: host 
+
+  app:
+    container_name: buildingmotif-app
+    depends_on: # require that the api server has started
+      - api
+    build: ../../../../buildingmotif-app
+    image: buildingmotif-app:latest
+    ports:
+      - target: 4200
+        published: 4200
+        protocol: tcp
+        mode: host
```

Comment:
we may want to merge our docker-compose fixtures at some point. Then we can have a subset of tests which need to run in that environment. This would allow us to use gh actions to handle uping and building the containers. And we could also do it once instead of redeploying them for every test.

Reply:
I like the idea, though to make each of the tests independent we will probably want to wipe any state anyway. Can you make an issue to handle this in the future?